dynamicweb/urls.py

```diff
@@ -1,8 +1,8 @@
 from django.conf.urls import patterns, include, url
```

Comment:
Lint's complaining that we are never using `patterns`. It is not being used, so we can remove that.